ui/components/ui/disclosure/disclosure.js

```diff
@@ -1,9 +1,48 @@
 import React, { useState, useRef, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import { Icon, IconName, IconSize } from '../../component-library';
+import { Icon, IconName, IconSize, Text } from '../../component-library';
+import { Color, TextVariant } from '../../../helpers/constants/design-system';
+import { DISCLOSURE_TYPES } from './disclosure.constants';
 
-const Disclosure = ({ children, title, size }) => {
+/**
+ * @param {string} type
+ * @param {string} title
+ * @returns {JSX.Element}
+ */
+const renderSummaryByType = (type, title) => {
```

Comment:
Can we change type to variant?
```suggestion
const renderSummaryByType = (variant, title) => {
```